tests/Utils/ObjectMixin.getMagicProperites().phpt

```diff
@@ -0,0 +1,117 @@
+<?php
+
+/**
+ * Test: Nette\Utils\ObjectMixin::getMagicProperties()
+ */
+
+use Nette\Utils\ObjectMixin;
+use Tester\Assert;
+
+
+require __DIR__ . '/../bootstrap.php';
+
+
+/**
+ * @property $getter
+ * @property $getter2 by ref
+ * @property $setter
+ * @property A\B $both with typehint
+ * @property $missing
+ * @property $Upper
+ * @property $cAse
+ * @property $private
+ * @property $protected
+ * @property $static
+ * @property-read $read
+ * @property-write $write
+ * @property-x $invalid
+ * abc @property $invalid2
+ */
+class TestClass
```

Comment:
ad 1) one test is enough, isn't it?
ad 2) hmmm, yes, it can be loosen always
ad 3) I dont like to support this